src/lib/ngsiNotify/Notifier.cpp

```diff
@@ -50,32 +50,302 @@ Notifier::~Notifier (void)
   LM_T(LmtNotImplemented, ("Notifier destructor is not implemented"));
 }
 
+
+
+/* ****************************************************************************
+*
+* substitute - 
+*/
+static void substitute(std::string* sP, const std::string& in, const ContextElement& ce)
+{
+  *sP = in;
+}
+
+
+
+/* ****************************************************************************
+*
+* templateNotify - 
+*/
```

Comment:
A little explanation in the function preamble would be greate... what this method does? build the notification payload after applyting template? Maybe I haven't reach the point in the PR where this gets clear, but a little explanation would help anyway.